stable/pomerium/values.yaml

```diff
@@ -0,0 +1,144 @@
+# For detailed explanation of each of the configuration settings see
+# https://www.pomerium.io/docs/config-reference.htmls
+
+# settings that are shared by all services
+config:
+  # secrets can be generated with `head -c32 /dev/urandom | base64`
+  sharedSecret: "LL3g/NoKkao7mTec2/wYSOZAki1VbR89q+kqbdFj0c4="
+  cookieSecret: "tAhd5A5Y8R/H/msFhkNdt8dCym2SatVbh4UrcDk3+Ms="
+  policyFile: |-
+    - from: httpbin.corp.pomerium.io
+      to: http://httpbin
+      allowed_domains:
+      - pomerium.io
+    - from: external-httpbin.corp.pomerium.io
+      to: httpbin.org
+      allowed_domains:
+      - gmail.com
+    - from: weirdlyssl.corp.pomerium.io
+      to: http://neverssl.com
+      allowed_users:
+      - bdd@pomerium.io
+      allowed_groups:
+      - admins
+      - developers
+    - from: hello.corp.pomerium.io
+      to: http://hello:8080
+      allowed_groups:
+      - admins
+tls:
+  generateTLS: true
+  certificateAuthority: ""
```

Comment:
@desimone this doesn't seem to be used anywhere…

Reply:
I'm a little stuck on what to do with ingress right now, which is what I was going to use this for if not using external certs.

Reply:
hehe, yes, usually the ingress is quite specific to each deployment. Personally I wrote a chart that depends upon the pomerium one and I generate the ingress as well as the policy file from a values file containing all the needed data to generate it :)